jenkins/clouds/aws/daily/policies/run_policies.py

```
@@ -60,10 +63,10 @@ def run_cmd(cmd: str):
     This method runs the shell command
     :param cmd:
     :type cmd:
-    :return:
-    :rtype:
+    :return: the raw os.system status (0 on success)
+    :rtype: int
     """
-    os.system(cmd)
+    return os.system(cmd)
```

Ruff (0.16.1)

[error] Starting a process with a shell, possible injection detected

(S605)